src/datasources/cache/cache.service.interface.ts

```diff
@@ -14,4 +14,6 @@ export interface ICacheService {
   deleteByKey(key: string): Promise<number>;
 
   deleteByKeyPattern(pattern: string): Promise<void>;
+
+  expire(key: string, expireTimeSeconds: number);
```

Comment:
nitpick: return type is missing here

Reply:
Fixed in [7538cd9](https://github.com/safe-global/safe-client-gateway/pull/828/commits/7538cd923d47b7eaefbfde53d1482a36af131894)